docs/manuals/core/scheduling/index.md

```diff
@@ -0,0 +1,278 @@
+# Taipy Scheduler
+
+!!! warning "Available in Taipy Enterprise edition"
+
+    This section is relevant only to the Enterprise edition of Taipy.
+
+This documentation focuses on providing necessary information to use the Taipy `Scheduler^`, and in particular
+the capabilities related to scenario management and job execution. It is assumed that the reader already knows the
+[Taipy Core entities](../entities/index.md) described in a previous chapter.
+
+Let’s assume also that the configuration represented in the picture below is implemented in the
+module <a href="./code_example/my_config.py" download>`my_config.py`</a>.
+
+=== "Module my_config.py"
+
+    ```python linenums="1"
+    {%
+    include-markdown "./code_example/my_config.py"
+    comments=false
+    %}
+    ```
+
+## Start/Stop the scheduler
+
+For the scheduler to run the scheduled methods, first you need to start the scheduler using the `start()` method.
```

Comment:
```suggestion
To make the scheduler execute the scheduled methods, you must first initiate the scheduler by using the `start()` method.
```

Reply:
I don't like the term `initiate` to talk about starting a service.
It s like a car you start it, you don't initiate it.

Reply:
OK let's keep "start" then :) But "initiate" is a common word used in IT writing when you want to start/begin a process.

Reply:
OK. My vocabulary is not so good. Let's use the most common word, whatever it is.

Reply:
But ways are OK from the English point of view, so we can keep "start" as well. The "start" wording is used quite often it other topics/sections as well, so to be consistent.